packages/main/src/themes/Button.css

```diff
@@ -12,7 +12,7 @@
 	font-size: var(--sapFontSize);
 	text-shadow: var(--_ui5_button_text_shadow);
 	border-radius: var(--_ui5_button_border_radius);
-	border-width: 0.0625rem;
+	border-width: var(--sapButton_BorderWidth);
 	cursor: pointer;
 	background-color: var(--sapButton_Background);
 	border: 1px solid var(--sapButton_BorderColor);
```

Comment:
Should not this:
_border: 1px solid var(--sapButton_BorderColor);_
become:
_border: var(--sapButton_BorderWidth) solid var(--sapButton_BorderColor);_

and perhaps then this becomes obsolete:
_border-width: var(--sapButton_BorderWidth);_

Reply:
Done.